docker/Dockerfile.windows.2004

```diff
@@ -0,0 +1,20 @@
+# escape=`
+
+FROM mcr.microsoft.com/windows/servercore:2004 AS git
+SHELL ["powershell.exe", "-Command", "$ErrorActionPreference = 'Stop'; $ProgressPreference = 'SilentlyContinue';"]
+
+RUN [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12 ; `
+    Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.28.0.windows.1/MinGit-2.28.0-64-bit.zip -OutFile git.zip; `
```

Comment:
The other Windows containers are using

```
Invoke-WebRequest -UseBasicParsing https://github.com/git-for-windows/git/releases/download/v2.21.0.windows.1/MinGit-2.21.0-64-bit.zip -OutFile git.zip;
```

Just making a note of that for @bradrydzewski